File: src/app/models/historique-login.ts
import { Utilisateur, UtilisateurDto } from './utilisateur';

export class HistoriqueLogin {
  id: number;
  action: string;
  createdDate: Date;

  utilisateur: Utilisateur;
}

export class HistoriqueLoginDto {
  id: number;
  action: string;
  createdDate: Date;

  utilisateurDto: UtilisateurDto;

}

File: src/main/java/com/chauffeur/models/HistoriqueLogin.java
package com.chauffeur.models;

import java.io.Serializable;
import java.util.Date;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;
import javax.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Table(name = "historiqueLogin")
@Data
@AllArgsConstructor
@NoArgsConstructor
public class HistoriqueLogin implements Serializable {
	
	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Long id;

	private Date createdDate;

	private String action;

	@ManyToOne
	@JoinColumn(name = "userId")
	private Utilisateur utilisateur;

}
